### PR TITLE
edit flexbox

### PR DESCRIPTION
**Pull request by FennyWilriani**
## General Checks

- [x] the branch is up to date with `main`/`master`
- [x] the code works when pulled and run locally
- [x] All CI checks pass
- [ ] all conflicts are resolved (if any)
- [x] PR has a descriptive title
- [x] PR has appropriate labels and milestones for easy identification
- [x] PR it is assigned to the owner
- [x] reviewers are assigned
- [x] the PR contributes only one focused change
- [ ] It is in the appropriate column in the project board (if necessary)
- [ ] has short and clear description
- [x] is linked to an issue (if it is related)
- [ ] feedback is addressed (if any and if it is appropriate feedback.)

## Markdown

- [x] the markdown source is formatted
- [x] spelling and grammar is correct in all text
- [x] The markdown looks correct when you preview the file
- [ ] all links and images work

### HTML

- [x] pages besides `/index.html` are stored in `/pages`
- [x] the code is well-formatted
- [x] the HTML is valid
- [ ] there are no inline styles (example: `style='color: red;'`)
- [ ] there are no `<style>` tags with CSS, all styles are hrefs
- [ ] there is no inline JavaScript (example: `onclick='doSomething()'`)
- [ ] there are no `<script>` tags with JS, all JS is in an separate file
- [ ] `id`s are used for JavaScript only, not for CSS
- [ ] semantic tags are used
- [ ] spelling and grammar is correct in all site content

### CSS

- [x] the code is well-formatted
- [x] passes all the linting checks

### CODE REVIEW

File: react/src/components/CarouselCards.jsx [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-    <Carousel>
-      {slider.map((item) => (
-        <Item key={item.id} item={item} onInput={onInput} />
-      ))}
-    </Carousel>
-  );
-}
+// export default function CarouselSlide({ onInput }) {
+//   return (
+//     <Carousel>
```
Comment (MMikhailova) [inline react/src/components/CarouselCards.jsx:8]: @FennyWilriani You commented CarouselCard element so it doesn't work.
Please check and make sure that all parts are working before you pull your changes.

**Review by MMikhailova — CHANGES_REQUESTED** [review]
@FennyWilriani Please check my comment for CarouselCard

File: react/src/components/CarouselCards.jsx (hunk 1 above)
Comment (FennyWilriani) [reply react/src/components/CarouselCards.jsx:8]: Yes, I had to comment on the updated code as I have an error displaying the result on react server, will fix it in the moment.

**Comment by FennyWilriani** [comment]
@MMikhailova should be ok now.

**Review by MMikhailova — APPROVED** [review]
@FennyWilriani looks good to me. Thank you

**Comment by minjupgeorge** [comment]
Looks great

**Pull request merged**